components/raftstore/src/store/config.rs

```diff
@@ -435,6 +445,9 @@ impl Default for Config {
             region_compact_redundant_rows_percent: None,
             pd_heartbeat_tick_interval: ReadableDuration::minutes(1),
             pd_store_heartbeat_tick_interval: ReadableDuration::secs(10),
+            // Disable full compaction by default
+            full_compact_tick_interval: ReadableDuration::secs(0),
+            full_compact_restrict_hours_local_tz: Vec::new(),
```

Comment:
Do we need to keep consitency with tidb usage such like `tidb_auto_analyze_start_time `?https://docs.pingcap.com/zh/tidb/stable/system-variables

Reply:
or maybe we can follow the naming of Linux `crontab`, such as `periodic-compact-crontab` ?

Reply:
> or maybe we can follow the naming of Linux `crontab`, such as `periodic_compact_crontab` ?

do we want to just use the name ending in `_crontab`, or follow the crontab string format?

Reply:
> Do we need to keep consitency with tidb usage such like `tidb_auto_analyze_start_time `?https://docs.pingcap.com/zh/tidb/stable/system-variables

proposal: i can change it to format used by this variable:
e.g., ``full-compact-start-times-times: [ "03:00 +0800", "23:00 +0800"]``

this will involve adding a new ``ConfigValue``: ``OffsetTime``:

```rust
full_compact_start_times: Vec<OffsetTime>, 
```

then by convention full compact will only look at hour part of it.

Reply:
Agree. The value format is more accessible to users.

And the previous suggestion from @bufferflies maybe a better choice than `crontab`.

Reply:
Done. /cc @bufferflies